doc/CODE_STYLE.md

```diff
@@ -1,41 +1,42 @@
-# Code Style Guide - EcoCode
-
-- [Introduction](#introduction)
-- [Source File Basics](#source-file-basics)
-	- [File encoding: UTF-8](#file-encoding-utf-8)
-	- [Indentation](#indentation)
-- [Source file structure](#source-file-structure)
-	- [Import statements](#import-statements)
-	- [Java source file organization](#java-source-file-organization)
-- [Formatting](#formatting)
-	- [Braces](#braces)
-		- [Block-like constructs: K\&R style](#block-like-constructs-kr-style)
-	- [Line wrapping](#line-wrapping)
-	- [Blank Lines](#blank-lines)
-- [Class declaration](#class-declaration)
-- [Naming](#naming)
-	- [Constant names](#constant-names)
-	- [Variable names](#variable-names)
-- [Programming Practices](#programming-practices)
-	- [File history](#file-history)
-	- [Organization of setter methods](#organization-of-setter-methods)
-	- [Ternary Operator](#ternary-operator)
-	- [Null Checks](#null-checks)
-	- [Use of @Override](#use-of-override)
-	- [Utility classes](#utility-classes)
-	- [Field and method references](#field-and-method-references)
-	- [Local variable type inference](#local-variable-type-inference)
-	- [Some rules from experience](#some-rules-from-experience)
-		- [Protections against ClassCast exceptions](#protections-against-classcast-exceptions)
-- [Javadoc](#javadoc)
-	- [Javadoc formatting](#javadoc-formatting)
-- [Tests](#tests)
-	- [Testing Framework](#testing-framework)
-	- [Naming](#naming-1)
-	- [Assertions](#assertions)
-	- [Mocking](#mocking)
-	- [Some rules from experience](#some-rules-from-experience-1)
-		- [Unit tests](#unit-tests)
+# Code Style Guide - Creedengo
+
+- [Code Style Guide - Creedengo](#code-style-guide---creedengo)
```

Comment:
sorry ... but the title is considered in table ... maybe change the style of title to only have real documentation section and not the title in table of contents